src/__tests__/views/signupView/__test__/signupView.test.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import { SignupView } from "../../../../views/signupView";
+
+const jwt = require("jsonwebtoken");
+
+const props = {
+  signupAction: jest.fn(),
+  toast: jest.fn(),
```

Comment:
Do we really have to define toast here as a prop seeing as its, not in the props on the view